docs(slider): update "Control externally" example to use <ButtonGroup />

2NNS-V opened the pull request:
<!---
Thanks for creating a Pull Request 💖!

Please read the following before submitting:
- PRs that adds new external dependencies might take a while to review.
- Keep your PR as small as possible.
- Limit your PR to one type (docs, feature, refactoring, ci, or bugfix)
- If a PR is not merged within one week of its creation, maintainers may intervene.
-->

Closes #3792 

## Description

update "Control externally" example to use <ButtonGroup />

koralle commented:
@2NNS-V 
As far as the content of your PR this time, `changesets` is not needed.
Could you please remove it?

2NNS-V commented:
Thank you for your feedback, @koralle!

I’ve made the requested updates:

Updated the documentation files:
- `/docs/contents/components/forms/slider/index.mdx`
- `/docs/contents/components/forms/slider/index.ja.mdx`
- Removed the changesets as they are not necessary for this PR.

Please let me know if there’s anything else I need to adjust! 😊

koralle commented:
@2NNS-V 
I have confirmed that the missing changes have been added.

Thank you!
